README.md

```diff
@@ -1,56 +1,70 @@
-# Syncron
+<div style="text-align: center;">
```

Comment:
It appears GitHub removes the style attribute... please change it back to `align="center"` instead :-)